### PR TITLE
use YQL - this is the future query language

### PR DESCRIPTION
**Pull request by kkraune**
- yes, query=bad looks better, but better get used to YQL from the start
  - rest of docs will be ported to YQL
- bonus: do not use -s when checking for 200 OK, better user experience

@jobergum

### CODE REVIEW

**Comment by bratseth** [comment]
I think "easy to get started" trumps "better to get used to X" (because there are so many X'es).

File: documentation/vespa-quick-start.html [hunk 1]
```diff
@@ -67,7 +67,7 @@
     <strong>Ensure the application is active:</strong><br>
     Wait for a valid <samp>200 OK</samp> response from:
 <pre>
-[root@vespa-container #/] curl -s --head http://localhost:8080/ApplicationStatus 
+[root@vespa-container #/] curl --head http://localhost:8080/ApplicationStatus
```
Comment (jobergum) [inline documentation/vespa-quick-start.html]: Why this change? -s hides the progress stuff ?

Comment (kkraune) [reply documentation/vespa-quick-start.html]: sorry my bad

**Comment by kkraune** [comment]
&gt; I think "easy to get started" trumps "better to get used to X"

Agreed - I will instead evaluate the next steps section and make sure documents there use YQL natively
